lib/src/signed_video_h26x_sign.c

```diff
@@ -489,6 +491,19 @@ SignedVideoReturnCode
 signed_video_add_nalu_for_signing(signed_video_t *self,
     const uint8_t *nalu_data,
     size_t nalu_data_size)
+{
+  return signed_video_add_nalu_for_signing_ts(self, nalu_data, nalu_data_size, NULL);
+}
+
+/**
+ * @brief Public signed_video_sign.h APIs
+ */
+
+SignedVideoReturnCode
+signed_video_add_nalu_for_signing_ts(signed_video_t *self,
```

Comment:
Rename signed_video_add_nalu_for_signing_ts to signed_video_add_nalu_for_signing_time_stamps or something similar. The code should be readable.